src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -86,32 +86,56 @@ def test_no_space_cont_create(self):
             "Pool %s query data: %s\n", self.pool.uuid, self.pool.query_data)
 
         # create a 2nd container now that pool is full
-        try:
-            self.log.info("creating 2nd container")
-            cont2 = DaosContainer(self.context)
-            cont2.create(self.pool.pool.handle)
-            self.log.info("created 2nd container")
+        # try writing to this second container which should
+        # fail after writing a few Kb of data as there should
+        # be some space emptied up due to aggregation and
+        # rdb log compression
+        self.log.info("creating 2nd container")
+        cont2 = DaosContainer(self.context)
+        cont2.create(self.pool.pool.handle)
+        self.log.info("created 2nd container")
+        self.log.info("opening container 2")
+        cont2.open()
 
-            self.log.info("opening container 2")
-            cont2.open()
+        written_data_cont2 = 0
+        self.log.info("writing more objects, write expected to fail "
+                      "after writing a few Kb")
+        for obj_sz in [10, 1]:
```

Comment:
Minor: Some comments here will give future testers to understand what is going on here. Why select object_size 10, 1? Why not a different number?